docs/getting-started.txt

```diff
@@ -1,300 +1,338 @@
+This exercise requires a Determined cluster with multiple GPUs.
 
-Determined can coordinate multiple GPUs to train a single trial more quickly (*distributed
-training*). Distributed training performs best with complex models; therefore, the simple model used
-in this example may not demonstrate the full benefits of using distributed training.
+Determined can coordinate multiple GPUs to train a single trial more quickly by using distributed
+training. Distributed training is best applied to complex models. The model used in this example
+does not fully demonstrate the benefits of distributed training but is still useful in showing how
+to work with additional hardware resources.
 
-With Determined, moving from single-GPU training to distributed training is as simple as changing a
-single setting in the experiment config file. Specifically, :ref:`resources.slots_per_trial
-<exp-config-resources-slots-per-trial>` controls the number of GPUs used to train a single trial. It
-is ``1`` by default; setting it to a larger value enables distributed training. If
-``slots_per_trial`` is larger than the number of slots on any single agent machine, you should set
+Moving from single-GPU training to distributed training involves changing a single
+:ref:`resources.slots_per_trial <exp-config-resources-slots-per-trial>` setting in the experiment
+``distributed.yaml`` configuration file, which is similar to the ``const.yaml`` file used in the
+previous example, except that a *resources* property is added with the ``resources.slots_per_trial``
+field is set to ``8`` to represent eight GPUs. The default ``slots_per_trial`` setting is ``1``. If
+``slots_per_trial`` is larger than the number of slots on an agent, set
 ``resources.slots_per_trial`` to a multiple of the number of GPUs in each machine in the cluster.
-For example, if your cluster has 8-GPU machines, you should use a value such as 8, 16, 24, etc. If
-using a Determined cluster deployed in the cloud, by default each agent will have eight GPUs.
+For example, if your cluster consists of 8-GPU machines, use a value that is a multiple of eight.
+For a Determined cluster deployed in the cloud, the default value for each agent is eight GPUs.
+
+#. Create the experiment using the distributed.yaml file:
+
+   .. code:: bash
+
+      det experiment create distributed.yaml .
+
+#. Enter the cluster address in the browser.
+
+#. Accept the default determined username with no password and click **Sign In** to view the
+   dashboard.
+
+#. Click the **Experiment** tile to view the experiment’s trial display.
 
-Below is the ``distributed.yaml`` file, which is very similar to the ``const.yaml`` file except we
-now set ``resources.slots_per_trial``.
+The loss curve is similar to the single-GPU experiment in the previous exercise but the time to
+complete the trial is reduced by about half.
 
-.. code:: yaml
+***************************************************
+ Improve Model Accuracy with Hyperparameter Tuning
```

Comment:
```suggestion
 Run a Hyperparameter Tuning job
```

This is not to teach readers to improve their model accuracy but only to teach them to use it with Determined.